plugins/out_stackdriver/stackdriver_conf.c

```diff
@@ -311,6 +311,14 @@ struct flb_stackdriver *flb_stackdriver_conf_create(struct flb_output_instance *
         ctx->trace_key = flb_sds_create(DEFAULT_TRACE_KEY);
     }
 
+    tmp = flb_output_get_property("spanid_key", ins);
```

Comment:
Can we name this `span_id_key` for consistency?

Reply:
Yes, I think I can.
Do I need to add and fix commits?

Reply:
Renamed `spanid_key` to `span_id_key` and recreate commit (ea5b9376077be1b65c729252f0591c1ec8dd678a)